universal-application-tool-0.0.1/app/services/program/predicate/Operator.java

```diff
@@ -2,25 +2,31 @@
 
 /** Represents a JsonPath operator (https://github.com/json-path/JsonPath#filter-operators). */
 public enum Operator {
-  ANY_OF("anyof"),
-  EQUAL_TO("=="),
-  GREATER_THAN(">"),
-  GREATER_THAN_OR_EQUAL_TO(">="),
-  IN("in"),
-  LESS_THAN("<"),
-  LESS_THAN_OR_EQUAL_TO("<="),
-  NONE_OF("noneof"),
-  NOT_EQUAL_TO("!="),
-  NOT_IN("nin"),
-  SUBSET_OF("subsetof");
+  ANY_OF("anyof", "is any of"),
```

Comment:
suggestion: "contains any of" - this one confuses me because it is true if there is an intersection between the set on the left and the set on the right. Not sure how to make this clear to the admin

Reply:
Talked offline about how this `anyof` operator really means "the intersection of the two sets is non-empty". I think "contains any of" works well enough for this for now. We can reword it if we get feedback that it's confusing.